plugin/helpers_bots.go

```diff
@@ -45,7 +117,10 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 	// If the bot has already been created, there is nothing to do.
 	if botIdBytes != nil {
 		botId := string(botIdBytes)
-		return botId, nil
+		setImagesErr := p.setIconAndProfileImageIfNotEmpty(botId, args.ProfileImagePath, args.IconImagePath)
+		if setImagesErr != nil {
+			return botId, errors.Wrap(setImagesErr, "Failed to set icon or profile image")
```

Comment:
Nit: Because we return a specific error message in `p.setIconAndProfileImageIfNotEmpty` we don't have to wrap it here. 1/5